docs/install/run-the-kit.md

```diff
@@ -20,7 +20,7 @@ Listening on port 3000 url: http://localhost:3000
 
 ## Check it works
 
-In your web browser, visit [http://localhost:3000](http://localhost:3000)
+In your web browser, visit <a href="http://localhost:3000" target="_blank">http://localhost:3000<span class="visuallyHidden"> (opens in a new window)</span></a>
```

Comment:
I think it does. I observed a user yesterday follow the link from the docs, thereby loosing the documentation which has more to say - and in the future might have 'next steps'. Better to open in a new window IMHO.